packages/fetchai/skills/confirmation_aw1/registration_db.py

```diff
@@ -45,6 +45,8 @@ def __init__(self, **kwargs):
             if custom_path is None
             else custom_path
         )
+        if not os.path.exists(os.path.dirname(os.path.abspath(self.db_path))):
+            raise ValueError(f"Path={self.db_path} not valid!")
```

Comment:
@lrahmani this will now throw before registration to SOEF...so we should no longer have this problem :)